CMake/lrs_options.cmake

```diff
@@ -10,6 +10,7 @@ option(HWM_OVER_XU "Send HWM commands over UVC XU control" ON)
 option(COM_MULTITHREADED "Set OFF to initialize COM library with COINIT_APARTMENTTHREADED (Windows only)" ON)
 option(BUILD_SHARED_LIBS "Build shared library" ON)
 option(BUILD_UNIT_TESTS "Build realsense unit tests. Note that when enabled, additional tests data set will be downloaded from a web server and stored in a temp directory" OFF)
+option(BUILD_LIVE_TESTS "Build realsense live tests. Note that when enabled, additional tests data set will be downloaded from a web server and stored in a temp directory" OFF)
```

Comment:
`Build the live-test.exe test used in Travis. Note that when enabled, additional test data set will be downloaded from a web server`

Reply:
Let's change it to `BUILD_LEGACY_LIVE_TEST`

Reply:
done